app/views/content/life-as-a-teacher/explore-subjects/geography/_article.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach geography</h2>
+      <p> Maths is critical to science, technology and engineering as well as being fundamental for most jobs.</p>
```

Comment:
Not sure the page is quite ready for review yet @joegibb - maths content throughout needs replacing!